gno.land/pkg/integration/testing_node.go

```diff
@@ -72,10 +83,14 @@ func TestingMinimalNodeConfig(t TestingTS, gnoroot string) *gnoland.InMemoryNode
 	genesis := DefaultTestingGenesisConfig(t, gnoroot, pv.GetPubKey(), tmconfig)
 
 	return &gnoland.InMemoryNodeConfig{
-		PrivValidator:    pv,
-		Genesis:          genesis,
-		TMConfig:         tmconfig,
-		GenesisTxHandler: gnoland.PanicOnFailingTxHandler,
+		PrivValidator: pv,
+		Genesis:       genesis,
+		TMConfig:      tmconfig,
+		DB:            memdb.NewMemDB(),
+		InitChainerConfig: gnoland.InitChainerConfig{
+			GenesisTxResultHandler: gnoland.PanicOnFailingTxResultHandler,
```

Comment:
```suggestion
			GenesisTxResultHandler: gnoland.WarnOnFailingTxResultHandler,
```

Still thinking about a testing_node; but i think we should use a WarnOnFailing in some configurations, like gnodev.

Reply:
This would change the behaviour of current testing nodes; so, Another Time:tm: ?